### DISCUSSION TITLE
bibMacros10 for WinEdt11

### DISCUSSION THREAD

**Discussion by GyorgySZEIDL** [WinEdt-Team/WinEdt-Support#8] [1.1 General Help and Discussion]
Dear WinEdt users,

I  have a simple question:

Can I ad bibmcros10 for WinEdt10
to WinEdt11? Following the usual
procedure.

Thank you for your kind reply in advance.

Sincerely yours
G. Szeidl

--------------
Gyorgy SZEIDL, DSc
Professor Emeritus
Institute of Applied Mechanics
University of Miskolc
H-3515 Miskolc-Egyetemváros
Hungary
E-mail: [gyorgy.szeidl@gmail.com](mailto:gyorgy.szeidl@gmail.com)
Home: 2500 Esztergom
Vár köz 8.
Hungary
---------------------

**Comment by Alex-Simonic** [comment]
There may be some glitches with the automatic install macro for this package. It uses MUI to provide some install options and depending on your display resolution this interface may not be usable (especially on high DPI). This has been fixed for the upcoming WinEdt 11.2, compiled with the new Delphi 12 and MUI now properly scales to high dpi.

Manual installation is always possible (for those who know how to add extra menus in their local settings). This applies to most contributions on winedt.org.

WinEdt.org has not been maintained for over 5 years and is quickly becoming obsolete. A new WinEdt-Org repository should be created on GitHub and users could collaborate on packages currently available on WinEdt.org and keep them alive and relevant. WinEdt 11.2 features new functionality (fetching files from the web, zipping and unzipping folders, and more may be added to make this easier and more accessible to simple users).

This is an open invitation to users with experience on GitHub to join and set up a better way to maintain such contributions on GitHub (one at a time). I may help with some issues but have no experience setting up GitHub code collaboration projects that could replace WinEdt.org (which was run as a one-man-show and will eventually be discontinued). If anyone is interested in joining for this purpose let us know (bibMacros could be the first contribution to test the idea and see if it will take off)...

**Reply by GyorgySZEIDL** [reply]
Thank you for the detailed explanation. I shall be waiting for WinEdt11.2. I hope it will be ready soon.
